### PR TITLE
Create xmlstarlet Galaxy tools

### PR DESCRIPTION
**Pull request by yvanlebras**
Following https://github.com/galaxyecology/tools-ecology/issues/51 and https://github.com/conda-forge/staged-recipes/pull/16458/files , here is a first proposal of Galaxy tool for xmlstarlet

### CODE REVIEW

**Comment by bgruening** [comment]
ping @yvanlebras

File: tools/xmlstarlet/xmlstarlet.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+**Inputs**
+
+A xml metadata file using a standard (for exemple EML, ISO19115,...) and a xsl file describing the mapping between the standard terms from input file and terms of the searched output standard.
+
+-----
+
+**Outputs**
+
+A xml metadata file using a new standard (for example ISO19115, EML, ...).
+   </help>
```
[the remark this answers is not in the document]
Comment (yvanlebras) [reply tools/xmlstarlet/xmlstarlet.xml:44]: DOne (I hope ;) )

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
> Input staging problem: Test input file (input_standardx.xml) cannot be found.

;)

**Comment by bgruening** [comment]
It was failing, left? So this will not be pushed to the TS.

**Comment by yvanlebras** [comment]
Arf... Can I reopen it and look at it more into details?

**Comment by bgruening** [comment]
Create a new one, manipulating a whitespace or something in the wrapper.

**Pull request merged**